addons/mail/static/src/components/chatter/chatter.js

```diff
@@ -103,6 +107,21 @@ class Chatter extends Component {
         this.chatter.update({ isComposerVisible: false });
     }
 
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onScrollPanelScroll(ev) {
+        //FIXME this is horrible.
+        if (!this._threadRef || !this._threadRef.comp || !this._threadRef.comp._messageListRef) {
+            return;
+        }
+        if (!this._threadRef.comp._messageListRef.comp) {
+            return;
+        }
+        this._threadRef.comp._messageListRef.comp._onScroll(ev);
+    }
```

Comment:
If this works and runbot is green we should just create public methods on the various children that abstract the private ref, similar to what do for focus, and then it would be good enough.

Reply:
How would you name those methods ? `scroll` ?

Reply:
`onScroll` maybe, the goal is not to scroll itself but to handle the event